.github/workflows/formatters.yaml

```diff
@@ -0,0 +1,30 @@
+name: Automatic Code Formatters
+on: push
```

Comment:
Also checking... This is intended to run on _every_ push? Not just pushes to the main branch?

Reply:
Yes, that was my intention. I want it to show up when someone pushes to their private branches as well, so they might notice the extra patch before sending a pull request.

Reply:
ACK. But I'm going to miss having `ninja test` clean up formatting _before_ pushing.

Reply:
Train yourself to use `ninja clang-format` and you'll be mostly fine. I don't want to implement a `python-black` runner mostly because different versions of it make different decisions. Hence why we're only running it once in github actions rather than on each distro like we did in Travis CI.